libs/template/templates/default-python/template/{{.project_name}}/resources/{{.project_name}}.pipeline.yml.tmpl

```diff
@@ -6,10 +6,11 @@ resources:
       {{- if or (eq default_catalog "") (eq default_catalog "hive_metastore")}}
       ## Specify the 'catalog' field to configure this pipeline to make use of Unity Catalog:
       # catalog: catalog_name
+      target: {{.project_name}}_${bundle.target}
       {{- else}}
       catalog: {{default_catalog}}
+      schema: {{.project_name}}_${bundle.target}
```

Comment:
We can actually leave `schema` outside of the `{{if}}`, since it's also supported with HMS pipelines.